content/docs/coldchain/equipment.md

```diff
@@ -0,0 +1,210 @@
+
+![Equipment nav](/docs/coldchain/images/equipment_route.png)
+
+#### Paginate Equipment
+
+The list can display a fixed number of assets per page. On the bottom left corner, you can see how many assets are currently displayed on your screen.
+
+![Page](/docs/distribution/images/os_list_showing.png)
+
+If you have more assets than the current limit, you can navigate to the other pages by tapping on the page number or using the right of left arrows (bottom right corner).
+
+![Page](/docs/distribution/images/os_list_pagenumbers.png)
+
+You can also select a different number of rows to show per page using the option at the bottom right of the page.
+
+![Rows per page](/docs/introduction/images/rows-per-page-select.png)
+
+#### Filter Equipment
+
+To add a filter to the page, choose the required filter from the drop down. Multiple filters can be combined.
+
+![Equipment filters](/docs/coldchain/images/equipment_filter.png)
+
+#### Delete Equipment
+
+Assets can be selected and deleted using the toolbar action:
+
+<p align="center">
+    <img src="/docs/coldchain/images/delete_rows.png" width="250" height="375">
+</p>
+
+### Importing And Exporting
+
+#### Import
+
+Cold chain assets can be imported from a csv file using the `Import` button
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_button.png" width="550" height="280">
+</p>
+
+This will open an import modal.
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_modal.png" width="750" height="420">
+</p>
+
+An example template comma-separated-value (csv) is available for download here:
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_template_download.png" width="750" height="420">
+</p>
+
+When importing assets from a store, they will be assigned to that store.
+When importing assets from the [central server](/docs/administration/synchronisation/), an additional field of `store` will be required which will assign assets to their respective stores.
+
+Information will need to be entered in the format provided by the csv template in order for Open mSupply to be able to process and upload this data.
+
+A csv file can be uploaded once it has been created in the example format.
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_upload.png" width="750" height="420">
+</p>
+
+#### Export
+
+Asset can be exported as a csv using the `Export` button
+
+<p align="center">
+    <img src="/docs/distribution/images/export.png" width="150" height="67">
+</p>
+
+### Create new asset
+
+Cold chain assets can be created manually with the `New Asset` button
+
+<p align="center">
+    <img src="/docs/coldchain/images/create_asset_button.png" width="150" height="55">
+</p>
+
+Clicking this button will open the `Create Asset` modal
+
+With this modal you can assign:
+
+- the Asset Category
+- the asset catalogue item the asset belongs to
```

Comment:
🤔 maybe 
```suggestion
- the asset catalogue item the Asset will be based on: this will link information from the catalogue item ( such as manufacturer & model ) to this newly created Asset
```